src/webgl/buffer.js

```diff
@@ -185,17 +185,56 @@ export default class Buffer extends Resource {
     return this;
   }
 
-  // WEBGL2 ONLY: Reads data from buffer into an ArrayBuffer or SharedArrayBuffer.
+  //
+  /**
+   * WEBGL2 ONLY: Reads data from buffer into an ArrayBufferView or SharedArrayBuffer.
+   *
+   * @param {ArrayBufferView} dstData= -ArrayBufferView to copy elements into,
```

Comment:
I personally don't think it makes sense to document a method twice. This is a public method and we have explicit docs in our markdown. We should focus on improving those public docs instead of duplicating text here. 

That's why there in general is only a one line comment on public methods. That's been the intention, obviously some code is older so the code may not be 100% consistent.